common/core/desktop/include/keyman/keyboardprocessor.h

```diff
@@ -659,6 +659,12 @@ typedef struct {
   km_kbp_option_item const * default_options;
 } km_kbp_keyboard_attrs;
 
+typedef struct {
+  km_kbp_virtual_key key;   // The key the belongs
+  uint32_t modifier_flag;   // modifier flag for the rule
```

Comment:
```suggestion
  km_kbp_virtual_key key;
  uint32_t modifier_flag;
```

I'm not sure that the line comments add relevant info? 😁